Playmate/Models/Friends Related/Invitation.m

```diff
@@ -0,0 +1,32 @@
+//
+//  Invitation.m
+//  Playmate
+//
+//  Created by Jocelyn Tseng on 7/21/22.
+//
+
+#import "Invitation.h"
+
+@implementation Invitation
+
+@dynamic invitationToId;
+@dynamic invitationFromId;
+@dynamic sessionObjectId;
+
++ (nonnull NSString *)parseClassName {
+    return @"Invitation";
```

Comment:
I was thinking if it's possible for your models to return the name of the class/object, maybe something like this 
`return NSStringFromClass ([Invitation class]);`
or similar if at all possible.